src/audio_worklet.js

```diff
@@ -168,7 +169,7 @@ class BootstrapMessages extends AudioWorkletProcessor {
         //
         // '_wsc' is short for 'wasm call', using an identifier that will never
         // conflict with user messages
-        messagePort.postMessage({'_wsc': d.callback, args: [d.contextHandle, 1/*EM_TRUE*/, d.userData] });
+        messagePort.postMessage({'_wsc': d.callback, args: [d.contextHandle, 1/*EM_TRUE*/, {{{ to64('d.userData') }}}] });
```

Comment:
Why is this needed?  I would hope that userData can be sent over as JS pointer (AKA a number) and never a bigint.

Reply:
In general it should be converted then the function is called, but maybe its not possible here since you don't know the function signature at the point of the call?

Reply:
The `wasmCall` above is used by both the message and by these collection of functions:

https://github.com/emscripten-core/emscripten/blob/052cd1a081d2194c1db4634e5f582e90e80f8750/src/lib/libwebaudio.js#L326

So exactly that, we don't know the signature, and this is used to broker all the different types of calls:

https://github.com/emscripten-core/emscripten/blob/052cd1a081d2194c1db4634e5f582e90e80f8750/src/audio_worklet.js#L173

Reply:
Left as-is and made a note.

Reply:
Ok, can we add a comment then?  Something like "Convert pointer argments here since we don't have enough information available at the actual callsite"

Reply:
Done.